src/bump-prerelease.ts

```
@@ -0,0 +1,153 @@
+    const remotes = stdout.split('\n').map(r => r.trim()).filter(Boolean);
+
+    if (remotes.includes('upstream')) {
+      return 'upstream';
+    }
+    if (remotes.includes('origin')) {
+      return 'origin';
+    }
+    throw new Error('No upstream or origin remote found');
+  } catch (error) {
+    if (error instanceof Error) {
+      throw new Error(`Failed to determine upstream remote: ${error.message}`);
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)